deepchecks/vision/checks/train_test_validation/train_test_label_drift.py

```diff
@@ -228,6 +229,11 @@ def compute(self, context: Context) -> CheckResult:
 
         return CheckResult(value=values_dict, display=displays, header='Train Test Label Drift')
 
+    def reduce_output(self, check_result: CheckResult) -> Dict[str, float]:
+        """Return label drift score per label property."""
+        drift_values = [label_property['Drift score'] for label_property in check_result.value.values()]
```

Comment:
why not
{name: label_property['Drift score'] for name, label_property in check_result.value.items()}